src/analysis/souffle/examples/multimic/multimic-variant.arcs

```diff
@@ -0,0 +1,69 @@
+// To be able to handle this use-case, since we don't have information about 
+// policies or who is even in the room until runtime, we need to evaluate these 
+// checks at runtime. The taint analysis could work similarly to the static one 
+// we have now, but it happens at runtime: reading from handles causes the tags 
+// on that handle to propagate to the write handles. Downgrades still remove 
+// tags.
+
+// Represents an actual microphone within a smart mic listening to audio.
+particle MicIn
+  audio: writes AudioData { }
+  // MicIn says hasTag("audio", unidentified_people::userSpeech)
+  // NOTE: at runtime, this also receives facts that add tags for the 
+  // identified users from "MobileDeviceRuntime" (see also comment in 
+  // multimic.authlogic).
+
+// Storage credentials can be in the form of authorization logic facts
+// and guarded by the may/will check. So I didn't include credentials checking 
+// here.
+
+particle Notifier
+  audioIn: reads AudioData { }
+  networkChannelsToUserDevices: writes Notification { }
```

Comment:
I was thinking, upon reviewing this line, that we should probably have an indication that this does not derive from anything, as it would likely be pre-generated, static strings. But I'm not sure if the current `derives from` syntax admits of an output that does not derive from any input. I guess that's another reason to switch to @bgogul's suggested `does not derive from` syntax.

Reply:
I thought derives from was more about removing edges between two handles from the graph. Isn't it enough to just not put any tags on the handle? Is the point to catch accidentally leaving out tags?